### DISCUSSION TITLE
[Weekly Review Report]: Jan 1 - 7

### DISCUSSION THREAD

**Discussion by IsabelWassink** [wikipathways/wikipathways-help#96] [curation-reports]
### Recent changes

 * Small changes to various pathways

### Under construction

 * Nothing over 30 days old

### New pathways

 * One new pathway: [WP5435](https://classic.wikipathways.org/index.php/Pathway:WP5435), which looks like a duplicate of [WP5420](https://classic.wikipathways.org/index.php/Pathway:WP5420). I don't see any differences between the two so possibly WP5435 could be deleted

### Deleted Pathways

 * 

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by danidi** [comment]
The pathways are not identical. I think this is related to the question of @alexmadsen1 on how to merge changes in a pathway here: https://github.com/wikipathways/wikipathways-help/discussions/95

**Comment by alexmadsen1** [comment]
[WP5435] is a accidental branch and can be killed.

**Reply by danidi** [reply]
Thank you for the confirmation. I've tagged the pathway for deletion.
